docs/content/installation/windows-service.en-us.md

```diff
@@ -36,7 +38,7 @@ If you use SQLite3, change the `PATH` to include the full path:
 PATH     = c:/gitea/data/gitea.db
 ```
 
-# Register as a Windows Service
+## Register as a Windows service
```

Comment:
Interesting that we have a `<h1>` and `<h2>` with the same text…

Reply:
Should enable https://github.com/DavidAnson/markdownlint/blob/main/doc/md024.md

Reply:
Actually the rule is partially enabled. It does allow same heading at different levels (`allow_different_nesting`). If I forbid that, it finds these:

```
docs/content/administration/cmd-embedded.en-us.md:104 MD024/no-duplicate-heading/no-duplicate-header Multiple headings with the same content [Context: "### Example"]
docs/content/administration/cmd-embedded.zh-cn.md:86 MD024/no-duplicate-heading/no-duplicate-header Multiple headings with the same content [Context: "### 示例"]
docs/content/installation/database-preparation.en-us.md:187 MD024/no-duplicate-heading/no-duplicate-header Multiple headings with the same content [Context: "### PostgreSQL"]
docs/content/installation/database-preparation.zh-cn.md:185 MD024/no-duplicate-heading/no-duplicate-header Multiple headings with the same content [Context: "### PostgreSQL"]
docs/content/installation/database-preparation.zh-cn.md:253 MD024/no-duplicate-heading/no-duplicate-header Multiple headings with the same content [Context: "### MySQL"]
```

Reply:
In any case, I think it should be forbidden because the anchors will not work correctly.

Reply:
It seems docasaurus actually can somewhat deal with it:

https://docs.gitea.com/administration/cmd-embedded#example
https://docs.gitea.com/administration/cmd-embedded#example-1

Only issue is if the first example is deleted, the link that previously pointed to it will now point to the other example.

Reply:
https://github.com/go-gitea/gitea/pull/27500 will fix the cases on main branch.